src/main/java/com/bluebreezecf/tools/sparkjobserver/api/SparkJobServerClientImpl.java

```diff
@@ -47,6 +32,10 @@
 import org.apache.http.impl.client.HttpClientBuilder;
 import org.apache.log4j.Logger;
 
+import java.io.*;
```

Comment:
Just follow the above imports and just import the real using package instead of *